test/src/org/labkey/test/tests/panoramapremium/TargetedMSSampleManagerIntegrationTest.java

```diff
@@ -88,17 +108,24 @@ public void testSampleTypeNavigation()
         assertElementPresent(Locator.linkWithText(s2));
         assertElementPresent(Locator.linkWithText(s3));
         clickAndWait(Locator.linkWithText(s1));
-        checker().verifyTrue("Sample link did not navigate to sample manager application",
-                getCurrentRelativeURL(false).contains(WebTestHelper.buildRelativeUrl("sampleManager", getProjectName() + "/" + Sample_Manager_Subfolder, "app")));
+        checker().fatal().verifyTrue("Sample link did not navigate to Sample Manager application",
+                getCurrentRelativeURL(false).contains(WebTestHelper.buildRelativeUrl("SampleManager", getProjectName() + "/" + Sample_Manager_Subfolder, "app")));
```

Comment:
Use advanced assertions library to produce more useful error messages.
```suggestion
        Assertions.assertThat(getCurrentRelativeURL()).as("Sample link navigation for Sample Manager product")
            .contains(WebTestHelper.buildRelativeUrl("SampleManager", getProjectName() + "/" + Sample_Manager_Subfolder, "app")));
```

Reply:
done